torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18269,7 +18265,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         "argsort",
         dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16),
         dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16),
-        sample_inputs_func=sample_inputs_argsort,
+        sample_inputs_func=sample_inputs_sort,
```

Comment:
argsort is the second value returned by torch.sort(). Same API as sort, not sure why samples weren't reused before.

```
torch.argsort(input, dim=- 1, descending=False, stable=False) → Tensor
   torch.sort(input, dim=- 1, descending=False, stable=False, *, out=None)
```

Note: on the aten side, stable has no default:
```
func: argsort.stable(Tensor self, *, bool stable, int dim=-1, bool descending=False) -> Tensor
```